### PR TITLE
[WIP] Adding tests for python

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- added python script to check flags and etc
- change search missing modules

### CODE REVIEW

File: tests/Python.Tests.ps1 [hunk 1]
```diff
@@ -22,10 +42,22 @@ Describe "Tests" {
     }
 
     if (IsNixPlatform $Platform) {
+
+        It "Check missing modules in docs" {
```
Comment (maxim-lobanov) [inline tests/Python.Tests.ps1]: `Check missing modules in build_output`

File: tests/Python.Tests.ps1 [hunk 2]
```diff
@@ -22,10 +42,22 @@ Describe "Tests" {
...
+            Write-Host "Search for missing modules..."
+            $ArtifactLocation = $env:BUILD_BINARIESDIRECTORY
+            $buildOutputLocation = Join-Path $ArtifactLocation "build_output.txt"
```
Comment (maxim-lobanov) [inline tests/Python.Tests.ps1]: `$buildOutputLocation = Join-Path $env:BUILD_BINARIESDIRECTORY "build_output.txt"`

File: tests/Python.Tests.ps1 (hunk 1 above)
Comment (dmitry-shibanov) [reply tests/Python.Tests.ps1]: Done.

File: tests/Python.Tests.ps1 (hunk 2 above)
Comment (dmitry-shibanov) [reply tests/Python.Tests.ps1]: Done.

**Pull request merged**